docs/config.rst

```diff
@@ -41,3 +41,20 @@ An environment variable like ``REACT_APP_FOO`` controls the setting ``FOO``.
     :default: ``https://${OIDC_DOMAIN}/userinfo``
 
    The audience for the access token that is generated by Auth0.
+
+
+.. envvar:: REACT_APP_REFRESH_AUTH_SESSION_SECONDS
```

Comment:
Perhaps specifically mentioning this is the period of the timer would help? Maybe something like `REACT_APP_AUTH_REFRESH_PERIOD_SECONDS`?